src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import { FilterType } from '../../types/FilterType';
+
+interface Props {
+  onChangeFilter: React.Dispatch<React.SetStateAction<FilterType>>;
+  currentFilter: FilterType;
+}
+
+export const TodoFilter: React.FC<Props> = ({
+  onChangeFilter,
+  currentFilter,
+}) => {
+  return (
+    <nav className="filter">
+      <a
```

Comment:
Please check [this article](https://medium.com/@h_locke/links-vs-buttons-e8b523660fb3) or any other on topic `button vs link philosophy`

Reply:
but in this case i have to navigate to /, /active, /completed in my URL, how i can do it without link?